diff -urNp setup-2.7.4-orig/bashrc setup-2.7.4/bashrc
--- setup-2.7.4-orig/bashrc	2008-10-10 18:29:39.000000000 +0200
+++ setup-2.7.4/bashrc	2008-11-14 09:37:24.000000000 +0100
@@ -18,14 +18,14 @@ if [ "$PS1" ]; then
 		if [ -e /etc/sysconfig/bash-prompt-xterm ]; then
 			PROMPT_COMMAND=/etc/sysconfig/bash-prompt-xterm
 		else
-	    	PROMPT_COMMAND='echo -ne "\033]0;${USER}@${HOSTNAME%%.*}:${PWD/#$HOME/~}"; echo -ne "\007"'
+			PROMPT_COMMAND='echo -ne "\033]0;${USER}@${HOSTNAME%%.*}:${PWD/#$HOME/~}"; echo -ne "\007"'
 		fi
 		;;
 	screen)
 		if [ -e /etc/sysconfig/bash-prompt-screen ]; then
 			PROMPT_COMMAND=/etc/sysconfig/bash-prompt-screen
 		else
-		PROMPT_COMMAND='echo -ne "\033_${USER}@${HOSTNAME%%.*}:${PWD/#$HOME/~}"; echo -ne "\033\\"'
+			PROMPT_COMMAND='echo -ne "\033_${USER}@${HOSTNAME%%.*}:${PWD/#$HOME/~}"; echo -ne "\033\\"'
 		fi
 		;;
 	*)
@@ -49,14 +49,18 @@ if ! shopt -q login_shell ; then # We're
 		fi
 	}
 
-	# Only run profile.d scripts if we are no login shell and interactive
-	if [ "$PS1" ]; then
-		for i in /etc/profile.d/*.sh; do
-			if [ -r "$i" ]; then
+	# Only display echos from profile.d scripts if we are no login shell
+    # and interactive - otherwise just process them to set envvars
+	for i in /etc/profile.d/*.sh; do
+		if [ -r "$i" ]; then
+			if [ "$PS1" ]; then
 				. $i
+			else
+				. $i &>/dev/null
 			fi
-		done
-	fi
+		fi
+	done
+
 	unset i
 	unset pathmunge
 fi
diff -urNp setup-2.7.4-orig/csh.cshrc setup-2.7.4/csh.cshrc
--- setup-2.7.4-orig/csh.cshrc	2008-10-10 18:29:39.000000000 +0200
+++ setup-2.7.4/csh.cshrc	2008-11-14 11:57:12.000000000 +0100
@@ -13,7 +13,7 @@ endif
 if ($?prompt) then
   if ($?tcsh) then
     set promptchars='$#'
-    set prompt='[%u@%m %c]%# ' 
+    set prompt='[%n@%m %c]%# ' 
     # make completion work better by default
     set autolist
   else
@@ -54,15 +54,17 @@ limit coredumpsize 0
 
 # Check if we aren't a loginshell and do stuff if we aren't
 if (! $?loginsh) then
-	if ($?prompt) then
-		if ( -d /etc/profile.d ) then
-	        	set nonomatch
-		        foreach i ( /etc/profile.d/*.csh )
-		                if ( -r $i ) then
-		                        source $i
-		                endif
-		        end
-		        unset i nonomatch
-		endif
-	endif
+    if ( -d /etc/profile.d ) then
+        set nonomatch
+        foreach i ( /etc/profile.d/*.csh )
+            if ( -r $i ) then
+                if ($?prompt) then
+                    source $i
+                else
+                    source $i >&/dev/null
+                endif
+            endif
+        end
+        unset i nonomatch
+    endif
 endif
diff -urNp setup-2.7.4-orig/csh.login setup-2.7.4/csh.login
--- setup-2.7.4-orig/csh.login	2008-10-10 18:29:39.000000000 +0200
+++ setup-2.7.4/csh.login	2008-11-14 09:49:42.000000000 +0100
@@ -17,18 +17,16 @@ if ( ! -f $HOME/.inputrc ) then
 setenv HOSTNAME `/bin/hostname`
 set history=1000
 
-if ( ! -f $HOME/.inputrc ) then
-	setenv INPUTRC /etc/inputrc
-endif
-
-if ($?prompt) then
-	if ( -d /etc/profile.d ) then
-	        set nonomatch
-	        foreach i ( /etc/profile.d/*.csh )
-	                if ( -r $i ) then
-	                        source $i
-	                endif
-	        end
-	        unset i nonomatch
-	endif
+if ( -d /etc/profile.d ) then
+        set nonomatch
+        foreach i ( /etc/profile.d/*.csh )
+                if ( -r $i ) then
+	                        if ($?prompt) then
+	                              source $i
+	                        else
+	                              source $i >& /dev/null
+	                        endif
+                endif
+        end
+        unset i nonomatch
 endif
diff -urNp setup-2.7.4-orig/profile setup-2.7.4/profile
--- setup-2.7.4-orig/profile	2008-10-10 18:29:39.000000000 +0200
+++ setup-2.7.4/profile	2008-11-14 12:07:36.000000000 +0100
@@ -39,20 +39,22 @@ if [ -x /usr/bin/id ]; then
 	MAIL="/var/spool/mail/$USER"
 fi
 
-HOSTNAME=`/bin/hostname`
+HOSTNAME=`/bin/hostname 2>/dev/null`
 HISTSIZE=1000
 
-if [ -z "$INPUTRC" -a ! -f "$HOME/.inputrc" ]; then
-    INPUTRC=/etc/inputrc
-fi
-
-export PATH USER LOGNAME MAIL HOSTNAME HISTSIZE INPUTRC
+export PATH USER LOGNAME MAIL HOSTNAME HISTSIZE
 
 for i in /etc/profile.d/*.sh ; do
     if [ -r "$i" ]; then
-    	. $i
+        if [ "$PS1" ]; then
+            . $i
+        else
+            . $i &>/dev/null
+        fi
     fi
 done
 
+export PATH
+
 unset i
 unset pathmunge
diff -urNp setup-2.7.4-orig/services setup-2.7.4/services
--- setup-2.7.4-orig/services	2008-10-10 18:29:39.000000000 +0200
+++ setup-2.7.4/services	2008-11-14 11:55:48.000000000 +0100
@@ -246,6 +246,8 @@ phonebook	767/tcp				# Network phonebook
 phonebook	767/udp
 rsync		873/tcp				# rsync
 rsync		873/udp				# rsync
+rquotad  875/tcp      # rquota daemon
+rquotad  875/udp      # rquota daemon
 telnets		992/tcp
 telnets		992/udp
 imaps	 	993/tcp				# IMAP over SSL
